R/dataprep_table_formatting.R

```diff
@@ -0,0 +1,62 @@
+#' format_table
+#' 
+#' This function takes a Sidora column-named table, and will re-format in 
+#' various ways for saving in different contexts.
+#' 
+#' @param table a sidora table
+#' @param format a type of format to convert to.
+#' 
+#' @examples
+#' \dontrun{
+#' sidora_table <- get_df("TAB_Site", con)
+#' format_table(sidora_table, "pandora_update_existing")
+#' }
+#' 
+#' @name format_table
+#' @export
+
+format_table <- function(sidora_table, format) {
```

Comment:
I like the versatile nature of this function!

Reply:
But now that I think about it... We don't have such wrapper functions nowhere else so far. We always call the relevant `as_...` functions directly. Do you think it's really simpler to introduce this concept now?

Reply:
My idea would be to do the same thing for all re-formatting (kable, tsv, markdown etc). 

The reason why I think this structure is better is it would be easier to remember one function and then use a 'help' message to give all the format types if you forget, rather than having to remeber each `as_*` function, which while their is autocomplete in Rstudio, not everyone will be using it and we could make `as_` to mean other things (at least in English it's not that spceific).